cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -440,7 +440,7 @@ func TestCompleteOptions(t *testing.T) {
 				ClusterGroups: map[string][]string{"group-a": {"default"}},
 				Secrets: []secretbootstrap.SecretConfig{{
 					From: map[string]secretbootstrap.ItemContext{"key-name-1": {Item: "item-name-1", Field: "field-name-1"}},
-					To:   []secretbootstrap.SecretContext{{Cluster: "default", Namespace: "ns", Name: "name"}},
+					To:   []secretbootstrap.SecretContext{{ClusterGroups: []string{"group-a"}, Cluster: "default", Namespace: "ns", Name: "name"}},
```

Comment:
Why did this need to change?

Reply:
(Was the previous test wrong? Do we need an additional test?)

Reply:
we are  intentionally retaining cluster_group info now in the `SecretContext`. This is so we can determine if it was originally part of a group or not. In the latter case we don't want to group it in the output. I don't _think_ retaining this info will cause any issues, but the test's expectation did need to change in order to specify that we do expect it to be there.